test/unit/plugins/reviewers.test.js

```diff
@@ -71,12 +74,69 @@ describe('ReviewersPlugin', function () {
     assume(reviewersPlugin.processesEdits).is.false();
   });
 
+  it('processes "ready for review" actions', function () {
+    assume(reviewersPlugin.processesReadyForReview).is.true();
+  });
+
   describe('.processRequest', function () {
     it('is a function', function () {
       assume(reviewersPlugin.processRequest).is.an('asyncfunction');
       assume(reviewersPlugin.processRequest).has.length(3);
     });
 
+    it('bails out if PR is a draft and requestForDrafts is false', async function () {
+      const draftContext = clone(mockContext);
+      draftContext.payload.pull_request.draft = true;
+
+      const requestReviewsStub = sandbox.stub(reviewersPlugin, 'requestReviews');
+      await reviewersPlugin.processRequest(draftContext, commenter, null);
+      assume(requestReviewsStub.called).is.false();
+      requestReviewsStub.restore();
```

Comment:
This belongs in an `after()` rather than being manually repeated at the end of every test.